app/config.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
   organisationApiUrl: process.env.ORGANISATION_API_URL || 'http://localhost:5103',
 
   // Buying Catalogue API
-  buyingCatalogueApiHost: process.env.API_HOST || 'http://localhost:5100',
+  solutionsApiUrl: process.env.SOLUTIONS_API_URL || 'http://localhost:5100',
```

Comment:
whats with the name change?

Reply:
the env_var `API_HOST` was too generic. I copied it from PublicBrowse.
and BUYING_CATALOGUE_API_URL was bit of a mouthful.

It is in theory the SOLUTIONS_API

I changed Host to Url to match the other.